tests/antithesis/test-template/robustness/main.go

```diff
@@ -97,8 +104,13 @@ func testRobustness(ctx context.Context, hosts, persistedRequestdirs []string, r
 	if err != nil {
 		panic(err)
 	}
+	serverDataPaths := make(map[string]string)
+	for i := range len(persistedRequestdirs) {
+		serverDataPaths[fmt.Sprintf("etcd%d", i)] = persistedRequestdirs[i]
+	}
```

Comment:
Please move it to separate function. We should try to simplify the robustness as possible.